mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -16,16 +16,6 @@ object English : LocalizedStrings {
   override val authenticateErrorFailure = "Something went wrong when authenticating"
   override val authenticateOr = "or"
 
-  override val boardColorBlack = "black"
```

Comment:
I think we need these strings for the game screen.

Reply:
Yes I rectified that